server/routerlicious/packages/lambdas/src/alfred/index.ts

```diff
@@ -619,6 +622,34 @@ export function configureWebSocketServices(
 				);
 			}
 
+			// Send signal to room from broadcast-signal endpoint
+			if (httpServer !== undefined && eventEmitter !== undefined) {
+				eventEmitter.on("broadcast-signal", (_, containerUrl) => {
+					const documentId = containerUrl.split("/")[containerUrl.split("/").length - 1];
+					const tenantId = containerUrl.split("/")[containerUrl.split("/").length - 2];
+					const roomFromBroadcastSignal: IRoom = { tenantId, documentId };
+
+					const signalMessageRuntimeMessage: ISignalMessage = {
+						clientId: null, // system signal
+						content: JSON.stringify({
+							type: "RuntimeMessage",
+							contents: {
+								content: {
+									type: "ExternalDataChanged",
+									content: "Data has changed upstream. Please import new data.",
+								},
+								type: "RuntimeMessage",
+							},
+						}),
+					};
+					socket.emitToRoom(
+						getRoomId(roomFromBroadcastSignal),
+						"signal",
+						signalMessageRuntimeMessage,
+					);
```

Comment:
Based on other usages of `socket.emitToRoom()` this seems to need a `.catch(...)`.

Reply:
+1